### ISSUE TITLE
Use separate face for timestamp-header-with-date vs timestamp-header

### ISSUE THREAD

**Issue by phil-s** [alphapapa/ement.el#91]
I find it useful when scanning the backlog if date changes jump out at me (often I'm checking for discussions from a particular day or range of days); but it's then distracting if every timestamp on account of `ement-room-timestamp-header-delta` has that same contrasting appearance.

I suggest an additional face for the date changes.  It can inherit the existing face by default, so it's a no-op for people who don't want these things to be different.

**Comment by alphapapa** [comment]
> I suggest an additional face for the date changes. It can inherit the existing face by default, so it's a no-op for people who don't want these things to be different.

> Perhaps faces should have their own customize group?

These seem like good ideas.  Would you be able to provide PRs for them?  (Assuming you've done the FSF copyright assignment.)

> Does the "with-date" header occur as soon as some event has passed the date boundary, or is it still necessary to have an elapsed ement-room-timestamp-header-delta number of seconds?

https://github.com/alphapapa/ement.el/blob/79e8cc51f104b35246e73c434a024f5635ddca5f/ement-room.el#L2675

Yes, it appears that the date-change header is only inserted when the delta is exceeded, which should be fixed.  I'll open an issue for that.

Thanks.

**Comment by phil-s** [comment]
> (Assuming you've done the FSF copyright assignment.)

Yes, I've assigned copyright for my Emacs contributions to the FSF.

**Comment by alphapapa** [comment]
> > (Assuming you've done the FSF copyright assignment.)
> 
> Yes, I've assigned copyright for my Emacs contributions to the FSF.

Ah, I didn't know that you are *the* Phil Sainty of emacs.git fame.  :)

Are you interested in writing a patch for this?  If you're not, I'll add it to the todo list.  Thanks.

**Comment by phil-s** [comment]
Notwithstanding that there's definitely no fame happening :) yes, that's me.

I'd like to contribute, but I'm struggling to make time for lots of things in general, so I think just add it to the todo list for now.
